Makefile

```
@@ -16,7 +16,8 @@
 
 .PHONY: test
 test:
-	go test -v -race -cover ./...
+	# Added MallocNanoZone env var, see: https://github.com/golang/go/issues/49138
```

not sure this was trying to keep a column width, but if so, would prefer to swap some url for context on monterey :)

```suggestion
	# MallocNanoZone env var avoids problems in macOS Monterey: golang/go#49138
```